examples/design-system/src/tokenami.css

```diff
@@ -202,28 +202,28 @@
     --color_white: #fff;
   }
 
+  @supports (color: color(display-p3 0 0 0)) {
+    :root, [data-theme="light"] {
+      --color_violet10: color(display-p3 .381 .306 .741);
+      --color_violet11: color(display-p3 .383 .317 .702);
+      --color_violet9: color(display-p3 .417 .341 .784);
+      --color_white: color(display-p3 1 1 1);
+    }
+  }
+
   [data-theme="dark"] {
     --color_violet10: #7d66d9;
-    --color_violet11: #baa7ff;
+    --color_violet11: #bba6ff;
```

Comment:
the base colours that lightning css generates are slightly different to the ones defined in the Radix palette, but i tested these and the difference is so minimal i'm not sure it's worth worrying about.